make/prepare

```diff
@@ -262,11 +277,17 @@ if customize_crt == 'on':
     if openssl_is_installed(shell_stat):
         private_key_pem = os.path.join(config_dir, "ui", "private_key.pem")
         root_crt = os.path.join(config_dir, "registry", "root.crt")
-        crt_conf_files = [ private_key_pem, root_crt ]
-        rmdir(crt_conf_files)
+        delfile(private_key_pem)
```

Comment:
since the key and cert files are moved to template directory, you should remove them from config directory either way.